packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.9.5
+
+* Return all the available cameras on iOS.
```

Comment:
Returns

See https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style